AIO/src/org/aio/activities/quests/RomeoAndJuliet.java

```diff
@@ -76,26 +74,22 @@ public void runActivity() throws InterruptedException {
                 break;
             case 50:
                 // Make sure we are not in the cut scene
-                if (Tab.INVENTORY.isDisabled(bot)) {
```

Comment:
We need the inventory tab check, if we only check for dialouges then in the cut scenes it will spam click stuff because it think its free.

Reply:
I've completed it, you are in a dialog while you are in the cutscene so yeah it works

Reply:
It will complete yes, but it will try to do things in cut scenes, like walking, trying to interact with things and so on.